src/main/java/com/dev/cinema/service/mapper/UserMapper.java

```diff
@@ -0,0 +1,12 @@
+package com.dev.cinema.service.mapper;
+
+import com.dev.cinema.model.User;
+import com.dev.cinema.model.dto.UserRequestDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper {
```

Comment:
we don't need UserMapper for now, let's remove it and add when we need it

Reply:
done